src/components/Button/Button.css

```diff
@@ -0,0 +1,5 @@
+.lqe__button--common {
```

Comment:
As this is a common button I would suggest removing the lqe from the class name as it will be used by more than just Life Quote Engine.